app/javascript/stylesheets/lesson-content.css

```diff
@@ -27,6 +27,38 @@
     @apply bg-gray-50 p-4 border-l-4 border-gold dark:bg-gray-700/40 dark:border-gold-700 rounded;
   }
 
+  .lesson-note > h4::before {
+    font-family: "Font Awesome 6 Free";
+    font-weight: 900;
+    font-size: large;
+    content: "\f249";
+    @apply mr-2 text-gold dark:text-gold-700;
+  }
+
+  .lesson-note--extra {
+    @apply border-purple-600 dark:border-purple-400;
+  }
+
+  .lesson-note--extra > h4::before {
+    font-family: "Font Awesome 6 Free";
+    font-weight: 900;
+    font-size: large;
+    content: "\f0eb";
+    @apply mr-2 text-purple-600 dark:text-purple-400;
+  }
+
+  .lesson-note--warning {
+    @apply border-red-600 dark:border-red-400;
```

Comment:
Nit: red-400 looks a little washed out to me on dark mode, red-500 has a better feel. Lets also up the light mode color to red-700 for a little bit of better contrast:

```suggestion
    @apply border-red-700 dark:border-red-500;
```

<img width="979" alt="Screenshot 2023-06-07 at 10 12 45" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/ecaf4fc4-4268-4836-872d-7cc77a51437c">

<img width="929" alt="Screenshot 2023-06-07 at 10 12 28" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/a7fe1e78-656b-4d9c-a77a-8880a8f40907">

Reply:
Looks nice to me!